openforcefield/tests/test_toolkits.py

```diff
@@ -498,6 +527,36 @@ def test_smiles_add_H(self):
         smiles2 = molecule.to_smiles(toolkit_registry=toolkit_wrapper)
         assert smiles2 == expected_output_smiles
 
+    @pytest.mark.skipif(not OpenEyeToolkitWrapper.is_available(), reason='OpenEye Toolkit not available')
+    def test_rdkit_from_smiles_hydrogens_are_explicit(self):
```

Comment:
I think you intended to check for the RDKit wrapper availability here.

Reply:
Good catch.